exchangeratesapi/api.py

```diff
@@ -72,6 +76,24 @@ def _check_date_format(self, date):
         if date:
             return datetime.strptime(date, self.DATE_FORMAT)
 
+    def _get_error_message(self, error):
```

Comment:
If you are not using `self` then this mean this is staticmethod:

```suggestion
    @staticmethod
    def _get_error_message(error):
```